src/sdk/pynni/nni/pbt_tuner/pbt_tuner.py

```diff
@@ -74,18 +74,16 @@ def exploit_and_explore(bot_trial_info, top_trial_info, factor, resample_probabi
     top_hyper_parameters = top_trial_info.hyper_parameters
     hyper_parameters = copy.deepcopy(top_hyper_parameters)
     random_state = np.random.RandomState()
+    hyper_parameters['load_checkpoint_dir'] = hyper_parameters['save_checkpoint_dir']
```

Comment:
just curious, why this tuner explicitly pass `checkpoint_dir` to trial? is it a part of the tuner algorithm?

Reply:
Got it, it is for weight sharing..